services/agent/requirements/_base.txt

```diff
@@ -41,16 +43,10 @@ click==8.1.3
     #   uvicorn
 dnspython==2.2.1
     # via email-validator
-email-validator==1.3.0
+email-validator==2.2.0
     # via pydantic
-fastapi==0.96.0
+fastapi==0.115.0
```

Comment:
OK so we also upgrade fastapi to the latest?

Reply:
Yup!